packages/jellyfish-testing/src/index.ts

```diff
@@ -81,3 +84,63 @@ class TestingJsonRpcClient extends JsonRpcClient {
     })
   }
 }
+
+export class TestingGroup {
+  private constructor (
+    public readonly group: ContainerGroup,
+    public readonly testings: Testing[]
+  ) {
+  }
+
+  static create (n: number, mnKeys?: MasterNodeKey[]): TestingGroup {
+    let containers: MasterNodeRegTestContainer[] = []
+    let testings: Testing[] = []
+
+    if (mnKeys !== undefined && mnKeys.length > 0) {
+      containers = mnKeys.map(k => new MasterNodeRegTestContainer(k))
+      testings = containers.map(c => Testing.create(c))
+    } else {
+      for (let i = 0; i < n; i += 1) {
+        const container = new MasterNodeRegTestContainer(GenesisKeys[i])
+        containers.push(container)
+
+        const testing = Testing.create(container)
+        testings.push(testing)
+      }
+    }
+
+    const group = new ContainerGroup(containers)
+
+    return new TestingGroup(group, testings)
+  }
```

Comment:
What about : 

```suggestion
  static create (n: number, Container: typeof MasterNodeRegTestContainer, mnKeys?: MasterNodeKey[]): TestingGroup {
    let containers: MasterNodeRegTestContainer[] = []
    let testings: Testing[] = []

    if (mnKeys !== undefined && mnKeys.length > 0) {
      containers = mnKeys.map(k => new Container(k))
      testings = containers.map(c => Testing.create(c))
    } else {
      for (let i = 0; i < n; i += 1) {
        const container = new MasterNodeRegTestContainer(GenesisKeys[i])
        containers.push(container)

        const testing = Testing.create(container)
        testings.push(testing)
      }
    }

    const group = new ContainerGroup(containers)

    return new TestingGroup(group, testings)
  }
```
to be able to call it as 
```
const group = TestingGroup.create(3, MultiOperatorGovernanceMasterNodeRegTestContainer, GenesisKeys)
```

Reply:
great.. its more configurable

Reply:
Better to use function, as TypeScript might rename the symbols

Reply:
the final output, `TestingGroup.create(3)` should be available for a test that does not care about container type and the keys.

Reply:
@Jouzo  i think i'm going backward :sweat_smile: .. btw @fuxingloh  helps solve the code ady.. thanks :pray: